.gitignore

```diff
@@ -35,6 +35,11 @@ var/
 *.manifest
 *.spec
 
+# pyenv
+#   For a library or package, you might want to ignore these files since the code is
+#   intended to run in multiple environments; otherwise, check them in:
+.python-version
```

Comment:
This file helps with my local setup, so excluding it from the repo here.

Reply:
Oooo I might add my own file to ignore in the future.